splunk-otel-android/src/main/java/com/splunk/rum/RumAttributeAppender.java

```diff
@@ -36,23 +36,16 @@ class RumAttributeAppender implements SpanProcessor {
 
     static final AttributeKey<String> SESSION_ID_KEY = stringKey("splunk.rumSessionId");
 
-    private final SessionId sessionId;
     private final VisibleScreenTracker visibleScreenTracker;
     private final ConnectionUtil connectionUtil;
 
-    RumAttributeAppender(
-            SessionId sessionId,
-            VisibleScreenTracker visibleScreenTracker,
-            ConnectionUtil connectionUtil) {
-        this.sessionId = sessionId;
+    RumAttributeAppender(VisibleScreenTracker visibleScreenTracker, ConnectionUtil connectionUtil) {
```

Comment:
`RumAttributeAppender` only has screen and network attributes now - in one of the following PRs I'll probably split it and have 2 separate `SpanProcessor`s for these instrumentations.